iceberg/iceberg-common/src/main/java/org/apache/gravitino/iceberg/common/IcebergConfig.java

```diff
@@ -64,7 +65,7 @@ public class IcebergConfig extends Config implements OverwriteDefaultConfig {
           .doc("The uri config of the Iceberg catalog")
           .version(ConfigConstants.VERSION_0_2_0)
           .stringConf()
-          .checkValue(StringUtils::isNotBlank, ConfigConstants.NOT_BLANK_ERROR_MSG)
+          .checkValue(TypeValidator::isJdbcURL, ConfigConstants.JDBC_URL_INVALID_ERROR_MSG)
```

Comment:
The IcebergConfig should not be changed, because the URI here might be another style (not JDBC url)